### PR TITLE
fixed Tim St. Clair's username

### PR DESCRIPTION
**Pull request by geekygirldawn**
His username was incorrect in this OWNERS file.

### CODE REVIEW

**Comment by geekygirldawn** [comment]
/cc @mrbobbytables 
Since I know you're also doing some OWNERS file cleanup :)

**Comment by feiskyer** [comment]
seems these are two different people from different companies. @timstclair @timothysc WDYT?

File: OWNERS [hunk 1]
```diff
@@ -9,7 +9,7 @@ approvers:
   - heartlock
   - mrunalp
   - saschagrunert
-  - timstclair
```
Comment (mrbobbytables) [inline OWNERS]: I think this is meant to be @tallclair - https://github.com/timstclair has a pointer to the newer GH handle.

Comment (geekygirldawn) [reply OWNERS]: Ah, good point ... the other Time St. Clair :) I've updated the commit with the correct GitHub handle.

**Comment by geekygirldawn** [comment]
OK, I've updated this with the username for the correct Tim St. Clair :)

**Comment by feiskyer** [comment]
@mrbobbytables @geekygirldawn thanks for the update
/lgtm
/approve

**Comment by tallclair** [comment]
Haha, thanks for this.

**Pull request merged**